pages/common/license.md

```diff
@@ -0,0 +1,19 @@
+# license
+
+> Create license files for open-source projects.
+
+- Create a license:
```

Comment:
Also, implicitness of author and year should probably be noted here. Does this work only for mercurial repos? What else does it support?